doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1427,9 +1485,14 @@ An `http` object contains the following subfields:
 
 + `url` : URL referencing the service which is invoked when a notification is generated. A NGSIv2
   compliant server must support `http` URL schema, other schemas could also be supported.
+
+An `httpExtended` object conatins the following subfields. If `httpExtened` is used, then
```

Comment:
These two phrases need to be switched. We can't end a sentence with 'contains the following subfields', without showing the list right after.

I propose:

    If the field `httpExtended` is used then Notification templates are supported (see specific section).
    An `httpExtended` object contains the following subfields:
    * 
    * ...